lib/diagnostic.rb

```diff
@@ -100,13 +103,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
```

Comment:
You'd want to use `.pop` here to remove from the end of an array. `arr.pop 2` would then remove the last 2 elements from `arr`